scripts/known_good.json

```diff
@@ -6,7 +6,7 @@
       "sub_dir": "Vulkan-Headers",
       "build_dir": "Vulkan-Headers/build",
       "install_dir": "Vulkan-Headers/build/install",
-      "commit": "v1.3.245"
+      "commit": "v1.3.246"
```

Comment:
You'll need to rebase on top of the [header update PR](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/pull/178) as soon as I get it merged.